services/ui/ws/window_tree.cc

```diff
@@ -929,6 +929,9 @@ void WindowTree::AddExternalModeWindowManagerState(
 }
 
 bool WindowTree::ShouldRouteToWindowManager(const ServerWindow* window) const {
+  if (window_server_->IsInExternalWindowMode())
+    return false;  // No window manager is available in external mode.
+
```

Comment:
Actually @msisov already removed the nullcheck. I'll move the check to IsWindowCreatedByWindowManager.